app/controllers/builds.js

```diff
@@ -13,7 +13,7 @@ export default Ember.Controller.extend({
     var id, number, type;
     id = this.get('repo.id');
     number = this.get('builds.lastObject.number');
-    type = this.get('tab') === "builds" ? 'push' : 'pull_request';
+    type = this.get('tab') === "builds" ? 'push' : this.get('tab').substr(0, this.get('tab').length-1);
```

Comment:
This change is inscrutable to me. My guess is that it’s about stripping a pluralisation? In cases like this, I would make it more explicit at the expense of being more verbose:

```
const tabName = this.get('tab');
const singularTab = tabName.substr(0, tabName.length);
```

Then people looking at the code don’t have to mentally run it to figure out what it’s for. This is my preferred approach for semi-opaque snippets of code that aren’t worth a block comment.